### PR TITLE
Add translated strings for timerange labels

### PR DESCRIPTION
**Pull request by JoelArmendariz**
Closes #

**Summary**

Need to translate time range labels

**Change List (commits, features, bugs, etc)**

 - Add to and display from the `i18n` default prop
 - Small fix to the dropdown items in the dataSeries where the options were unnecessarily taking the labels as the `text` prop instead of the dataSourceId.

**Acceptance Test (how to verify the PR)**

Go to any `DashboardEditor` story, add or select a timeseries card and ensure that the labels for the time range appear correctly.

### CODE REVIEW

File: src/components/CardEditor/CardEditForm/CardEditFormContent.jsx [hunk 1]
```diff
@@ -155,8 +173,8 @@ const CardEditFormContent = ({
 
   const validTimeRanges = getValidTimeRanges
     ? getValidTimeRanges(cardConfig, selectedDataItems)
-    : Object.keys(defaultTimeRangeOptions);
-
+    : defaultTimeRangeOptions;
+  console.log('i18n', mergedI18n);
```
Comment (tay1orjones) [inline src/components/CardEditor/CardEditForm/CardEditFormContent.jsx:177]:
```suggestion
```

**Comment by scottdickerson** [comment]
@JoelArmendariz I think you might need to keep the data item labels for those in the Summary dashboard view that you're trying to 'simplify' the names of

**Comment by JoelArmendariz** [comment]
> @JoelArmendariz I think you might need to keep the data item labels for those in the Summary dashboard view that you're trying to 'simplify' the names of

@scottdickerson The labels will still show under the dropdown where the user can edit each dataItem. UX decided that the dropdown would be a "source of truth" and exclusively show the dataSourceIds, while the list under it would show the user created labels, even if the names don't match up between the list and the dropdown.

**Review by scottdickerson — APPROVED** [review]
LGTM

**Review by bjornalm — APPROVED** [review]

**Pull request merged**